src/router/storeSync/LayerParamConfig.class.js

```diff
@@ -170,7 +171,7 @@ async function getAndDispatchFeatures(to, featuresPromise, store) {
             if (!query.z) {
                 await store.dispatch('zoomToExtent', {
                     extent: extent,
-                    maxZoom: 8,
+                    maxZoom: STANDARD_ZOOM_LEVEL_1_25000_MAP,
```

Comment:
I'm wondering if it would not make more sense to use here `from.query.z` instead of `to.query.z` ? This could avoid layer parameter order dependencies.

Reply:
we do not have access to 'from' here without changing every storesync `setValueInStore` function. Luckily, the 'to.query' is not modified at this state. The issue was only the maxZoom being modified by the zoomToExtent function